.renvignore

```diff
@@ -1 +1,2 @@
 inst/extdata/r_packages.R
+data-raw
```

Comment:
Ask renv to ignore packages we use in data-raw/ -- as it is a non-standard directory that faces only developers. Developers are expected to have usethis as part of their devleopment environment -- but we do not need to force users to do so.